src/components/ClusterSpinUp.js

```diff
@@ -1,17 +1,19 @@
 import React, {Component} from 'react';
 
 class ClusterSpinUp extends Component {  
-    clusterLinks = (url) =>   {
+    getUrlVars = (vars) => {
```

Comment:
Is this parsing the query string in the URL? If yes, I would personally use a library like [qs](https://www.npmjs.com/package/qs) for this

Reply:
And thus begins the npm module dependency nightmare :-) I'll have a look at it, obviously all I did was rename a variable!